airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/Rollout.yaml

```diff
@@ -0,0 +1,25 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/Rollout.yaml
+title: Rollout
+description: configuration for the rollout of a connector
+type: object
+additionalProperties: false
+properties:
+  initialPercentage:
```

Comment:
@clnoll I was wondering if we plan to version control the rollout progress. Shall we introduce a `currentPercentage` which would be updated via automated PRs by the orchestrator?

Reply:
@alafanechere I wasn't planning to. I'd prefer to keep the flow of information in one direction, and encourage people to use information from the RC table as the point of truth.

Reply:
🆗